### ISSUE TITLE
Including gtest.h fails on OS X

### ISSUE THREAD

**Issue by pgorczak** [ros/diagnostics#37]
While looking for the tuple class it somehow [assumes that it should include `<tr1/tuple>`](../blob/indigo-devel/self_test/gtest-1.6.0/include/gtest/gtest.h#L1561) although Clang implements C++11 with tuple being a part of the standard library (so it should be just `<tuple>`).

Sorry for not gathering more info from the header file for now, I will look into it when I have time.

**Comment by garaemon** [comment]
gtest-1.7.0 works on my environment (yotemite, llvm 3.5)

**Comment by FBaralli** [comment]
I have gtest 1.7.0 installed with brew but diagnostic_aggregator seems to be using its own gtest (1.6.0). At least that is what looks like from the error I get:
```
ros_catkin_ws/src/diagnostics/diagnostic_aggregator/gtest-1.6.0/include/gtest/gtest.h:1561:13: fatal error: 'tr1/tuple'
      file not found
#   include <tr1/tuple>  // NOLINT
            ^
1 error generated.
```

**Comment by garaemon** [comment]
@FBaralli 
Could you try this PR?
https://github.com/ros/diagnostics/pull/39

**Comment by trainman419** [comment]
@FBaralli as per the recommended gtest practice, diagnostics includes and builds its own copy of gtest. It doesn't use the system version.

**Comment by trainman419** [comment]
I've upgraded gtest to 1.7.0 in the hydro-devel branch. Please test and verify that it works before I do a release.

**Comment by pgorczak** [comment]
Thanks, I just tested it. I can confirm that 03aa72b76ba1add715c4d56a9e963b33c05821a9 fixes the issue on Yosemite+Indigo.

**Comment by trainman419** [comment]
Released into Hydro and Indigo as version 1.8.7
